File: Day2:Arrays/Duplicate_In_Array.py
from os import *
from sys import *
from collections import *
from math import *

def findDuplicate(arr:list, n:int):
    # Write your code here.
    # Returns an integer.
    arr.sort()
    ans = 0;
    for i in range(n):
        if arr[i] == arr[i+1]:
            ans =arr[i]
            break
    return ans

File: Day2_Arrays/Duplicate_In_Array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findDuplicate = function(nums) {
    let size = nums.length;
    let ans;
    nums.sort((a,b)=> a-b);
    for(let i=0;i<size-1;i++){
        if(nums[i]===nums[i+1])
            return nums[i];
    }
};
